components/blitz/src/omero/cmd/graphs/DuplicateI.java

```diff
@@ -414,7 +414,7 @@ public Object apply(Object original) {
                         }
                     }
                     /* process property values that do not relate to edges in the model object graph */
-                    for (final String property : graphPathBean.getSimpleProperties(superclassName)) {
+                    for (final String property : graphPathBean.getSimpleProperties(superclassName, false)) {
```

Comment:
I feared that it might be misleading: this current change makes one think about if one wanted nested or not. Depending on what extended metadata consumers need it might be worth splitting out a bunch of these methods by nested-or-not (some now conflate the two); it's mainly just this component-properties issue I have in mind with the "design work" of the PR description. (From memory, I think it mostly crops up for units and details.)